WTWSMS/common/religions/00_religions.txt

```diff
@@ -4775,6 +4801,19 @@ bosporan_group = {
 		can_grant_divorce = yes
 		can_call_crusade = yes
 		can_grant_invasion_cb = invasion
+
+		short_reign_opinion_year_mult = 0 # No short reign penalty
+
+		character_modifier = { 
+			fertility = 0.10
+			learning = 2
+			ai_honor = 1
+		}
+	
+		defensive_attrition = yes
+		independence_war_score_bonus = 50
+		attacking_same_religion_piety_loss = yes
+		has_heir_designation = yes
```

Comment:
Simple buff for Hypsistarians. The defensive attrition is for obvious reasons (they were difficult to drive out of the Byzantine Empire and it gives them a fighting chance). Attacking the same religion subtracts piety because of the small and close-knit nature of the Hypsistarian faithful. Independence war score bonus simply allows them, like pagans, to break away and protect their faith.